corehq/ex-submodules/soil/tests/test_progress_manager.py

```diff
@@ -0,0 +1,136 @@
+import mock
+from django.test import SimpleTestCase
+from freezegun import freeze_time
+
+from soil.progress import TaskProgressManager
+
+
+@mock.patch.object(TaskProgressManager, '_set_task_progress')
```

Comment:
Goes with previous suggestion in _corehq/ex-submodules/soil/progress.py_
```suggestion
import soil.progress as mod
from soil.progress import TaskProgressManager


@mock.patch.object(mod, 'set_task_progress')
```

Reply:
Thanks this is exactly what I was looking for, but for some reason I didn't think that worked. I'll merge this and try it out in a separate PR. Thanks for the tip!